### ISSUE TITLE
YouTube iframe forces keyboard users to tab through all suggested videos

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4109]
## Area or Component
Static site - support homepage

## Quick Summary
The embedded YouTube video on the page is part of the keyboard navigation, and tabbing through requires users to go through all the suggested videos.

### How to reproduce
1. Navigate to simplereport.gov/support
2. Start playing the YouTube video and pause
3. Tab from the top of the page and see all the suggested videos that appear

## Accessibility Violation
There's no way to skip past all this suggested content, making navigating the rest of the page after the video difficult.

## Action Requested
If possible, removed these suggested videos entirely: https://www.200ok.nl/tips/youtube-embed/

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1832428693&range=10:10

**Comment by emyl3** [comment]
@emmastephenson I was looking into way to disable this and all the answers I found said there used to be a parameter `rel` that you could set to `0` you could disable those suggested videos from displaying.

Unfortunately, after September 25, 2018 they changed that feature so if you set `rel` to `0` it will still show suggested videos but they will be from your channel.

https://developers.google.com/youtube/player_parameters#Parameters

An option could be looping through the video so it doesn't ever show those suggested videos when the video ends. (this can be accomplished via `loop=0` param)

There is no way to use the youtube `iframe` without having the suggested videos appear.

Should I look into alternative ways to embed videos or should we consider a different way to present this information without embedding a link?

**Comment by emmastephenson** [comment]
@emyl3 thanks for looking into this! I unfortunately think there's not too much else to be done here 😞 We can't really control what YouTube puts in their iframe viewer, and it sounds like there's not enough granular control here to change the settings meaningfully.

The good (?) news is that most users will have undoubtedly come across a YouTube iframe before, so this is probably not unexpected behavior. The Deque guidelines also state that:

> When iframes contain content from third-party web sites — as is often the case — it is not always possible to control the heading hierarchy. Because of the lack of control in these situations, the guidelines don't strictly require the heading hierarchies of the two documents to match, but it would still be better if they did.

(source: https://dequeuniversity.com/class/semantic-structure/iframes/semantic-structure-across-iframes)

With that in mind, I think it's ok to close this ticket.
